### PR TITLE
[modular] Renames the code for Inquisition ERT from Spanish Red to just Unexpected because admins keep asking me why I'm sending red alert ERTs (I'm not)

### PR DESCRIPTION
**Pull request by Odairu**
## About The Pull Request
Does what the title says

## Why It's Good For The Game
Differentiates it more clearly from actual red alert ERTs

## Proof Of Testing
Screenshot_12

## Changelog

:cl:
qol: its now easier to tell what ERT is being spawned
/:cl:

### CODE REVIEW

**Comment by BurgerLUA** [comment]
I thought Spanish Red was creative :( darn

**Review by projectkepler-ru — APPROVED** [review]

**Comment by Odairu** [comment]
> I thought Spanish Red was creative :( darn

Its very creative, but the last two times I've spawned it, Asay immediately lights up with "YOU SPAWNED A CODE RED?!?"

**Comment by destrucktoid** [comment]
Why not just name it "Spanish Inquisition" instead of "Spanish red" if we're making a monty python reference? Heck, might as well just make it a code "Unexpected" ERT. Since nobody expects the Spanish inquisition.

**Comment by projectkepler-ru** [comment]
> Why not just name it "Spanish Inquisition" instead of "Spanish red" if we're making a monty python reference? Heck, might as well just make it a code "Unexpected" ERT. Since nobody expects the Spanish inquisition.

you're right

File: modular_zubbers/code/datums/ert.dm [hunk 1]
```diff
@@ -15,7 +15,7 @@
 	code = "Gold"
 
 /datum/ert/inquisition
-	code = "Spanish Red"
+	code = "Inquisition"
```
Comment (projectkepler-ru) [inline modular_zubbers/code/datums/ert.dm:18]:
```suggestion
	code = "Unexpected"
```
Funnier

**Review by projectkepler-ru — CHANGES_REQUESTED** [review]
Not funny enough

**Pull request merged**